actions/ql/src/Security/CWE-275/MissingActionsPermissions.ql

```diff
@@ -14,13 +14,36 @@
 
 import actions
 
-from Job job
+Step stepInJob(Job job) { result = job.(LocalJob).getAStep() }
+
+bindingset[fullActionSelector]
+string versionedAction(string fullActionSelector) {
```

Comment:
This feels like it should be exposed in the library class for a uses-based step. Do we not have the ability to get the unversioned action name there already?

Reply:
Action selectors can actually take [a variety of forms](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsuses). I opted for a simple approach here; for a library we might want more full support. But I can see it would be nice to push it into the library.

Reply:
Actually, I can just use what is in `UsesStep.getCallee` (which is the same as here).